tests/test_validate_rules.py

```diff
@@ -0,0 +1,57 @@
+# SPDX-License-Identifier: GPL-3.0-or-later
+from copy import deepcopy
+
+import pytest
+from pydantic import ValidationError
+from pytest import raises
+
+from retasc.validator.models import Rule
+from retasc.validator.validate_rules import validate_rule
+
+
+def test_rule_valid(valid_rule_file):
+    try:
+        validate_rule(str(valid_rule_file))
+    except ValidationError:
+        assert False, "A valid rule dict should not raise ValidationError"
```

Comment:
This is an unused function now. Can you also replace `--cov=src` in `tox.ini` with `--cov=.` so the unused code gets detected even in the tests?